### PR TITLE
Fill vlan's interface list when calling get_vlan or get_vlans on cisco.

### PR DESCRIPTION
**Pull request by fbouliane**

### CODE REVIEW

File: netman/adapters/switches/cisco.py [hunk 1]
```diff
@@ -89,6 +89,7 @@ def get_vlan(self, number):
             vlan,
             self.ssh.do("show running-config interface vlan {} | begin interface".format(number))
         )
+        apply_interfaces_data(vlan, self.get_interfaces())
```
Comment (lindycoder) [inline netman/adapters/switches/cisco.py]: That can really slow the hell down this method that should be quick
Any other way not listing all interfaces?

**Comment by fbouliane** [comment]
Will check with network for alternative.

**Comment by emmurd** [comment]
Reopen to be consistent with the merged code.
It's the only way to get all trunked vlans for each interfaces

**Comment by lindycoder** [comment]
ok :+1:

**Pull request closed without merging**